charts/kargo/values.yaml

```diff
@@ -93,7 +93,7 @@ api:
     ## @param api.adminAccount.password A password for the admin account. Ignored if `passwordHash` is set. It is suggested that you generate this using a password manager or a command like: `openssl rand -base64 29 \| tr -d "=+/" \| cut -c1-25`.
     password: admin
     ## @param api.adminAccount.tokenSigningKey Key used to sign ID tokens (JWTs) for the admin account. It is suggested that you generate this using a password manager or a command like: `openssl rand -base64 29 \| tr -d "=+/" \| cut`
-    tokenSigningKey:
+    tokenSigningKey: ""
```

Comment:
What do you think about using a default value like `change me`?

I don't think the API server can actually handle this being blank.

Reply:
`change me` would likely have the same effect, so I'm on board. But I have tested with `tokenSigningKey: ''`, and the API server works for me.

Reply:
Ooooh.

I just remembered that this exists: http://masterminds.github.io/sprig/flow_control.html

For things where we don't want to provide a default and we _insist_ on the user providing their own value, this could be helpful.